src/main/java/com/easypost/model/Shipment.java

```diff
@@ -1063,4 +1062,28 @@ public Rate lowestRate(final List<String> carriers, final List<String> services)
 
         return lowestRate;
     }
+
+    /**
+     * Get rate for a shipment.
+     *
+     * @return Shipment object
+     * @throws EasyPostException
+     */
+    public Shipment getShipmentRate() throws EasyPostException {
```

Comment:
This doesn't just return one rate, it returns a list of rates so it should be pluralized. This is true of all the `getShipmentRate` occurrences. I'm also torn on the name of this. In our other libraries we call it `getRates` but we obviously have the getter already titled `getRates`... I need to think on this for a second.